scripts/globals/items/bottle_of_clerics_drink.lua

```diff
@@ -0,0 +1,62 @@
+-----------------------------------
+-- ID: 5395
+-- Item: bottle_of_clerics_drink
+-- Item Effect: Instantly removes all negative status effects from nearby party members
+--              Does not remove Plague or Curse
+-----------------------------------
+require("scripts/globals/settings")
+require("scripts/globals/msg")
+require("scripts/globals/item_utils")
+-----------------------------------
+local item_object = {}
+
+item_object.onItemCheck = function(target)
+    return 0
+end
+
+item_object.onItemUse = function(target)
+    local effects =
```

Comment:
Since this pattern is repeated, it should most likely be declared elsewhere and utilized.